src/Magento/FunctionalTestingFramework/Composer/Objects/ComposerFactory.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\FunctionalTestingFramework\Composer\Objects;
+
+use Composer\IO\BufferIO;
+
+class ComposerFactory
```

Comment:
I'm not 100% sure we need this class. Only use I found of `create()` is in `AbstractComposerHandler`. We should just call `\Composer\Factory::create(new BufferIO(), $this->composerFile)` directly instead of creating this factory class for a single usage.

Reply:
Removed the factory class